app/src/main/kotlin/com/rockspin/bargainbits/data/models/daos/GameDealDao.kt

```diff
@@ -0,0 +1,21 @@
+package com.rockspin.bargainbits.data.models.daos
+
+import android.arch.persistence.room.Dao
+import android.arch.persistence.room.Insert
+import android.arch.persistence.room.OnConflictStrategy
+import android.arch.persistence.room.Query
+import com.rockspin.bargainbits.data.models.GameDeal
+import io.reactivex.Flowable
+
+/**
+ * Created by valentin.hinov on 29/05/2017.
+ */
+@Dao
+interface GameDealDao {
```

Comment:
The other class is named `DbDealsDao` because it uses `DbDeals` objects. I agree the DbDeals name is not great but I was struggling to figure out what to name it